python/cuml/test/dask/test_base.py

```diff
@@ -21,6 +21,68 @@
 
 from cuml.dask.datasets import make_blobs
 
+import cuml
+import numpy as np
+import pytest
+
+from numpy.testing import assert_equal
+
+from cuml.dask.linear_model import LinearRegression
+
+from cuml.dask.datasets import make_regression
+from sklearn.model_selection import train_test_split
+
+
+def make_dataset(datatype, nrows, ncols, n_info):
+    X, y = make_regression(n_samples=nrows, n_features=ncols,
+                           n_informative=n_info, random_state=0)
+    X = X.astype(datatype)
+    y = y.astype(datatype)
+    X_train, X_test, y_train, y_test = train_test_split(X, y, train_size=0.8)
+    return X_train, y_train, X_test
+
+
+@pytest.mark.parametrize('datatype', [np.float32, np.float64])
+@pytest.mark.parametrize('keys', [cuml.dask.linear_model.LinearRegression])
+@pytest.mark.parametrize('data_size', [[500, 20, 10]])
+@pytest.mark.parametrize('fit_intercept', [True, False])
+def test_get_combined_model(datatype, keys, data_size, fit_intercept,
+                            client):
+
+    nrows, ncols, n_info = data_size
+    X_train, y_train, X_test = make_dataset(datatype, nrows,
+                                            ncols, n_info)
+    model = LinearRegression(fit_intercept=fit_intercept,
+                             client=client)
+    model.fit(X_train, y_train)
+
+    combined_model = model.get_combined_model()
+    assert combined_model.coef_ is not None
+    assert combined_model.intercept_ is not None
+
+
```

Comment:
Do we have anything that tests attribute proxying? like calling `coef_` on a dist model to confirm it's pulling correctly in the fit case and behaving correctly in the unfit case (which I guess would through an AttributeError?)

Reply:
Yeah, That's what `test_getattr` is testing.

Reply:
I realized I was missing the case before fit was called. Added those test cases. Also changed `ValueError` to `AttributeError`.